src/daemon/variables_entrypoint.sh

```diff
@@ -67,6 +67,7 @@ fi
 : "${MGR_IP:=0.0.0.0}"
 : "${CEPH_ARCH:=$(uname -m)}"
 : "${MON_PORT:=6789}"
+: "${PUBLIC_BIND_ADDRESS:=${PUBLIC_BIND_ADDRESS}}"
```

Comment:
This should not be needed since you check the variable in start_mon.sh

Reply:
This has only been added to keep the "LIST OF ALL VARIABLES" up-to-date. Can easily be removed if desired.